bsconfig.json

```diff
@@ -16,6 +16,10 @@
         // [
         //     "reason/bin/ppx_lwt"
         // ],
+        "js-post-build" : 
+        {
+            "cmd" : "/bin/cat"
```

Comment:
As discussed offline, this is gonna be slightly troublesome... The location differs for these commands, e.g. "we have different Reason setups; it doesn't matter where `foo` is as long as it's in PATH"

I'm regretting suggesting `foo` for `node_modules/foo`. Maybe it should be `@foo` like you first suggested.